xrdp/lang.c

```diff
@@ -226,22 +226,35 @@ int APP_CC
 get_keymaps(int keylayout, struct xrdp_keymap *keymap)
 {
     int fd;
+    int shortKeylayout; /* lower 16 bits of keylayout */
```

Comment:
We don't do camelCase for variables. Also, it's better to declare the variable in the block where it's used. I would call it "basic_key_layout". "short" is a C concept, not a problem domain concept.